### ISSUE TITLE
Too many banned IP

### ISSUE THREAD

**Issue by Azard** [nimiq-network/mining-pool#7]
It seems the ban metrics has some problems, many normal miners are banned.

**Comment by Bhlynd** [comment]
Can confirm, I am getting the same issue.

**Comment by christiangrieger** [comment]
Yes after every pool restart many many many users get banned, no way to get this solved and no way to hold a stable pool up that way. This problem need attention and should be fixed soon.

**Comment by riptl** [comment]
@ChristianGrieger I can implement a `shutdown` message that the pool sends to all connected miners so they stop working and submitting shares until the pool is up again.
